package-lock.json

```diff
@@ -492,9 +492,9 @@
       "dev": true
     },
     "@types/node": {
-      "version": "13.1.1",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-13.1.1.tgz",
-      "integrity": "sha512-hx6zWtudh3Arsbl3cXay+JnkvVgCKzCWKv42C9J01N2T2np4h8w5X8u6Tpz5mj38kE3M9FM0Pazx8vKFFMnjLQ==",
+      "version": "14.14.37",
```

Comment:
I think this is causing the process triggers tsc error.

Reply:
Forgot why I add this, but let me try to revert it. ಠ_ಠ

Reply:
Looks like you didn't need it?